lib/ui/page/home/page/my_profile/controller.dart

```diff
@@ -182,6 +179,12 @@ class MyProfileController extends GetxController {
   /// Returns the current [Credentials].
   Rx<Credentials?> get credentials => _authService.credentials;
 
+  /// Private flag for returning early from [onObserve] callback
+  bool _isIgnoreObserving = false;
+
+  /// Last selected [ProfileTab]
+  ProfileTab? _lastSelectedProfileTab;
```

Comment:
Looks like this variable is used only in the `onInit` method. If it stays this way, then it might be defined as a local variable in the method instead of a private field in the class.

Reply:
It's just a metter of habit, i think you right, it would be better to move declaration to method, i'll correct it

Reply:
moved it before getters as in Contribution Guide, can't move it to Init method cause onObserve now is function getter.

fixed on 3bdb4c364a9c19145ac3970c4389cdc034f8f30f commit